sdk/eventhub/azure-eventhub/azure/eventhub/aio/_eventprocessor/event_processor.py

```diff
@@ -215,6 +216,20 @@ def _create_tasks_for_claimed_ownership(
                 )
             await self._event_handler(partition_context, event)
 
+    async def _clean_receive(self, partition_context):
```

Comment:
I think this should be renamed to something like `_close_consumer`

Reply:
Changed the method name to _close_consumer. Could you re-approve?